# Default initial positions for simple_manipulator's ros2_control fake system

initial_positions:
  joint1: 0
  joint2: 0